src/credentials/attestation_verifier/DefaultDeviceAttestationVerifier.h

```diff
@@ -57,7 +57,11 @@ class CsaCdKeysTrustStore : public WellKnownKeysTrustStore
 class DefaultDACVerifier : public DeviceAttestationVerifier
 {
 public:
-    DefaultDACVerifier(const AttestationTrustStore * paaRootStore) : mAttestationTrustStore(paaRootStore) {}
+    DefaultDACVerifier(const AttestationTrustStore * paaRootStore,
+                       DeviceAttestationRevocationDelegate * revocationDelegate = nullptr) :
```

Comment:
Instead of having a default parameter, just add a second constructor with both arguments. This will keep backwards compatibility.

Reply:
Added one more constructor.